lib/widgets/poll.dart

```diff
@@ -69,46 +84,68 @@ class _PollWidgetState extends State<PollWidget> {
-            Expanded(
+      return Row(
+        crossAxisAlignment: CrossAxisAlignment.baseline,
+        textBaseline: localizedTextBaseline(context),
+        children: [
+          GestureDetector(
+            // TODO: Implement feedback when the user taps the button
+            onTap: () => _toggleVote(option),
+            behavior: HitTestBehavior.translucent,
+            child: ConstrainedBox(
+              constraints: const BoxConstraints(minWidth: 44, minHeight: 44),
+              child: Padding(
+                // For accessibility, the touch target is padded to be larger
+                // than the vote count box.  Still, we avoid padding at the
+                // start because we want to align all the poll options to the
+                // surrounding messages.
+                padding: const EdgeInsetsDirectional.only(
+                  end: 5, top: verticalPadding, bottom: verticalPadding),
+                child: Container(
+                  // Inner padding preserves whitespace even when the text's
+                  // width approaches the button's min-width (e.g. because
+                  // there are more than three digits).
+                  padding: const EdgeInsets.symmetric(horizontal: 4),
+                  decoration: BoxDecoration(
+                    color: theme.colorPollVoteCountBackground,
+                    border: Border.all(color: theme.colorPollVoteCountBorder),
+                    borderRadius: BorderRadius.circular(3)),
+                  child: Center(
+                    child: Text(option.voters.length.toString(),
+                      style: textStyleBold.copyWith(
+                        color: theme.colorPollVoteCountText, fontSize: 20))))))),
+          Expanded(
+            child: Padding(
+              // This padding and the bottom padding on the vote count box
+              // extends the row by the same extent. This ensures that there
+              // still will be consistent spacing between rows, when the text
+              // takes more vertical space than the vote count box.
+              padding: const EdgeInsets.only(bottom: verticalPadding),
               child: Wrap(
                 spacing: 5,
                 children: [
                   Text(option.text, style: textStyleBold.copyWith(fontSize: 16)),
                   if (option.voters.isNotEmpty)
                     // TODO(i18n): Localize parenthesis characters.
                     Text('($voterNames)', style: textStyleVoterNames),
-                ])),
-          ]));
+                ]))),
+        ]);
     }
 
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        Padding(padding: const EdgeInsets.only(bottom: 6), child: question),
+        Padding(
+          // `verticalPadding` out of 6 logical pixels come from the first
+          // option row.
+          padding: const EdgeInsets.only(bottom: 6 - verticalPadding),
```

Comment:
> Ah, I guess if the text is long, then there's still top padding because CrossAxisAlignment.baseline means the option text starts at the same level as the count and expands down from there, and so the button still extends further upward from the baseline than the text does.

Yeah, exactly. I've made them symmetric in the new revision.